rotkehlchen/db/upgrades/v50_v51.py

```diff
@@ -18,7 +18,9 @@
 def upgrade_v50_to_v51(db: 'DBHandler', progress_handler: 'DBUpgradeProgressHandler') -> None:
     """Upgrades the DB from v50 to v51. This happened in the v1.42 release."""
 
-    @progress_step(description='Rename event_identifier column to group_identifier in history_events table.')  # noqa: E501
+    @progress_step(
+        description='Rename event_identifier column to group_identifier in history_events table.',
```

Comment:
no need to do this change. If it has a noqa: E501 it's fine. We do this quite often